src/Uno.UI/UI/Xaml/Controls/MediaPlayer/MediaPlayerElement.net.cs

```diff
@@ -1,4 +1,4 @@
-﻿#if !HAS_UNO_WINUI
+﻿#if !__WASM__ && !HAS_UNO_WINUI
```

Comment:
I think this should be redundant. `.net.cs` files are only included for net461. So we already know it's not Wasm.

```suggestion
﻿#if !HAS_UNO_WINUI
```